main/nav_create_simulated_model.py

```
@@ -0,0 +1,1739 @@
+        )
+        fl.addRow('V:', v_spin)
+        u_spin = QDoubleSpinBox()
+        u_spin.setRange(0.0, 10000.0)
+        u_spin.setDecimals(1)
+        u_spin.setValue(p.get('u', 0.0))
+        u_spin.valueChanged.connect(
+            lambda v, i=idx: self._on_star_field(i, 'u', v)
+        )
+        fl.addRow('U:', u_spin)
+        # Keep references so drag updates can sync the UI
+        setattr(w, 'v_spin', v_spin)
```

Ruff (0.14.6)

Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)